### ISSUE TITLE
Make blog title and subtitle editable

### ISSUE THREAD

**Issue by the0and1** [Mvzundert/mvz_blog#8]
Because now it always says "Marius van Zundert's blog - A clean blog written in Laravel 5.1".

**Comment by Mvzundert** [comment]
Agreed! For now it's locatedin a config file at: https://github.com/Mvzundert/mvz_blog/blob/master/config/blog.php
